conmon-rs/server/src/pause.rs

```diff
@@ -39,11 +42,8 @@ pub struct Pause {
 /// The global shared multiple pause instance.
 static PAUSE: OnceCell<Pause> = OnceCell::new();
 
-/// The global path for storing bin mounted namespaces.
-const PAUSE_PATH: &str = "/var/run/conmonrs";
-
-/// The file path for storing the pause PID.
-const PAUSE_PID_FILE: &str = ".pause_pid";
+/// The global path for storing namespaces.
+const DEFAULT_PATH: &str = "/var/run";
```

Comment:
@haircommander is it fine to have it set to `/var/run`? I don't see many reasons for making it configurable (yet).

Reply:
the one thing it would be useful for is testing. For instance, in cri-o's `namespaces.bats` we use it to easily check

Reply:
if it were my preference, we would do it in one fell swoop so we don't need to introduce more client-side knowledge of what options the server supports

Reply:
Updated the PR to make it configurable.